docs/ref/models/querysets.txt

```diff
@@ -2194,10 +2194,35 @@ query. The default is to create all objects in one batch, except for SQLite
 where the default is such that at most 999 variables per query are used.
 
 On databases that support it (all but Oracle), setting the ``ignore_conflicts``
-parameter to ``True`` tells the database to ignore failure to insert any rows
-that fail constraints such as duplicate unique values. Enabling this parameter
-disables setting the primary key on each model instance (if the database
-normally supports it).
+Setting the ``ignore_conflicts`` parameter to ``True`` tells the database to
+ignore failure to insert any rows that fail constraints such as duplicate unique
+values.
+
+Setting the ``update_conflicts`` parameter to ``True``,
+``unique_fields`` parameter to which fields are unique and
+``update_fields`` parameter to which fields you want to update,
+while on conflict it will update ``update_fields`` by ``unique_fields``.
+
+* If you use mysql as your database,
```

Comment:
Please use official names in docs:
- `mysql` -> `MySQL`,
- `postgresql` -> `PostgreSQL`.